### DISCUSSION TITLE
Paid account for more bandwith and space, but ignored by GitHub?

### DISCUSSION THREAD

**Discussion by Tobias1967** [community/community#136852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello!
I took a paid account for more bandwith and space, but it seems, that this issue is ignored by Github and I still get this error message:

"2024-08-26 15:01:06.266 [info] batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access."

That´s weired!

**Comment by ebndev** [comment]
Hi @Tobias1967,

The best place to go for answers to billing / account-related questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Other GitHub users may have run into the issue before, but the Support team will be the ones to access specific details about your situation.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
